Nos/Service/FeatureFlags.swift

```diff
@@ -1,17 +1,25 @@
 import Foundation
 import Dependencies
 
-/// The set of feature flags used by the app.
-protocol FeatureFlags {
+/// Enum to represent each feature flag. Add new feature flags here easily.
```

Comment:
I don't think the phrase "enum to represent" adds much here, and Apple's documentation doesn't do that. Maybe simplify it to something like:
```suggestion
/// Feature flags for enabling experimental or beta features.
```